src/libpriv/rpmostree-importer.cxx

```diff
@@ -636,6 +624,10 @@ compose_filter_cb (OstreeRepo         *repo,
 
   gboolean error_was_set = (error && *error != NULL);
 
+  /* Sanity check that path is absolute */
+  if (path == NULL || *path != '/')
+    return OSTREE_REPO_COMMIT_FILTER_SKIP;
```

Comment:
Previously a non-absolute path triggered an assertion, right?  I think it'd be better to make it a hard error instead of a silent skip.   I believe it's a "shouldn't happen" case.

Reply:
I think we can keep doing `g_assert (path != NULL)` because that definitely shouldn't happen.